docs/from_the_sources.md

```diff
@@ -103,13 +105,35 @@ That's it! Go on your browser to the xo-server IP address, and it works! :)
 
 ## Misc
 
-- You can also consider using [forever](https://github.com/nodejitsu/forever) to have always the process running.
+- You can use [forever](https://github.com/nodejitsu/forever) to have the process always running:
 
 ```
 $ yarn global add forever
 $ forever start bin/xo-server
 ```
 
+- Or you can use  [forever-service](https://github.com/zapty/forever-service) to install XO as a service, so it starts automatically at boot. As root:
```

Comment:
Is it "standard" enough? Why this particular software?

Reply:
it certainly seems to be the most popular way of making node apps act like a service and start on boot. Trying to help people get it running at boot so they don't revert to using the "xo install scripts" floating around to do so because they always break